api/app_analytics/influxdb_wrapper.py

```diff
@@ -61,8 +62,8 @@ def add_data_point(self, field_name, field_value, tags=None):
     def write(self):
         try:
             self.write_api.write(bucket=settings.INFLUXDB_BUCKET, record=self.records)
-        except HTTPError:
-            logger.warning("Failed to write records to Influx.")
+        except (HTTPError, InfluxDBError) as e:
+            logger.warning("Failed to write records to Influx: %s", str(e))
```

Comment:
How about 

```suggestion
            logger.warning(
                "Failed to write records to Influx: %s",
                str(e),
                exc_info=e,
            )
```
?

Reply:
Oh nice, yeah, I didn't know about that 👍